rollup.npm.js

```diff
@@ -149,28 +140,22 @@ export default [
   // Browser minified version.
   {
     input: 'packages/main.js',
-    globals: { react: 'React' },
     external: ['react'],
-    output: [{ file: pkg.unpkg, format: 'umd' }],
-    name: 'BoardgameIO',
+    output: [
+      {
+        file: pkg.unpkg,
+        format: 'umd',
+        name: 'BoardgameIO',
+        globals,
+      },
+    ],
     plugins: plugins.concat([
       builtins(),
-      commonjs(),
       resolve({ browser: true, preferBuiltins: false }),
+      commonjs(),
       replace({
         'process.env.NODE_ENV': JSON.stringify('production'),
       }),
-      uglify(
```

Comment:
Missed this. Why are we no longer minifying the code here?

Reply:
I'm guessing you deleted this because it was generating some errors and forgot to add it back. 
Added `rollup-plugin-terser` instead.

Reply:
Ah yeah, I removed it to simplify things. At first it was having some errors so I upgraded it but then the config for it was broken. `rollup-plugin-terser` looks a better solution anyway, and zero config 💯